### DISCUSSION TITLE
Support Zero Copy Send

### DISCUSSION THREAD

**Discussion by robd003** [Kludex/uvicorn#2050] [Ideas]
Could we get uvicorn to support Zero Copy Send?

https://asgi.readthedocs.io/en/latest/extensions.html#zero-copy-send

**Comment by Kludex** [comment]
What's your use case for it?

**Reply by robd003** [reply]
I'd like to use it for serving static files more efficiently.

**Comment by synodriver** [comment]
You can merge it manually since it's already exists [here](https://github.com/encode/uvicorn/pull/1210)
